### PR TITLE
Add test coverage for BZ:1931489

### PR DESCRIPTION
**Pull request by Gauravtalreja1**
### Problem Statement
Missing coverage for BZ:1931489, where we verified timeout to kill/execution_timeout_interval setting is honored by long running ansible-playbook job

### Solution
Add test coverage for BZ:1931489, to verify long running ansible-playbook job along with timeout to kill/execution_timeout_interval setting

### Dependent PRs:
https://github.com/SatelliteQE/nailgun/pull/1139

### CODE REVIEW

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_ansible.py::TestAnsibleREX::test_negative_ansible_job_timeout_to_kill
nailgun: 1139

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6895
Build Status: UNSTABLE
PRT Comment: pytest tests/foreman/api/test_ansible.py::TestAnsibleREX::test_negative_ansible_job_timeout_to_kill --external-logging
Test Result : ================= 1 failed, 13 warnings in 1307.98s (0:21:47) ==================
```

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_ansible.py::TestAnsibleREX::test_negative_ansible_job_timeout_to_kill
nailgun: 1139

**Comment by Satellite-QE** [comment]
**PRT Result**
```
Build Number: 6897
Build Status: SUCCESS
PRT Comment: pytest tests/foreman/api/test_ansible.py::TestAnsibleREX::test_negative_ansible_job_timeout_to_kill --external-logging
Test Result : ================= 1 passed, 13 warnings in 1233.29s (0:20:33) ==================
```

**Review by Griffin-Sullivan — APPROVED** [review]
Looks good to me. Acked the nailgun PR too.

File: tests/foreman/api/test_ansible.py [hunk 1]
```diff
@@ -470,3 +470,72 @@ def test_positive_ansible_job_on_multiple_host(
...
+            ---
+            - name: Sample Ansible Playbook to pause
+              hosts: all
+              gather_facts: no
+              tasks:
+              - name: Pause for 5 minutes
+                pause:
+                  minutes: 5
+        '''
+        result = rhel_contenthost.register(
```
Comment (shweta83) [inline tests/foreman/api/test_ansible.py:506]: Can we use api method to register here?

**Review by shweta83 — APPROVED** [review]
Ack, pending one non-blocker comment.

File: tests/foreman/api/test_ansible.py (hunk 1 above)
Comment (Gauravtalreja1) [reply tests/foreman/api/test_ansible.py:506]: there are other tests using it, I'd consider changing it in separate PR if required

**Comment by Gauravtalreja1** [comment]
Just resolved the merge conflict, PRT passed in previous comment with build: 6897

**Review by shubhamsg199 — APPROVED** [review]

**Pull request merged**